Jellyfin.Api/Helpers/DynamicHlsHelper.cs

```diff
@@ -710,7 +745,19 @@ private string ReplacePlaylistCodecsField(StringBuilder playlist, StringBuilder
             return oldPlaylist.Replace(
                 oldValue.ToString(),
                 newValue.ToString(),
-                StringComparison.OrdinalIgnoreCase);
+                StringComparison.Ordinal);
+        }
+
+        private string ApplyFlacCaseWorkaround(StreamState state, string srcPlaylist)
+        {
+            if (!string.Equals(state.ActualOutputAudioCodec, "flac", StringComparison.OrdinalIgnoreCase))
+            {
+                return string.Empty;
+            }
+
+            var newPlaylist = srcPlaylist.Replace(",flac\"", ",fLaC\"", StringComparison.Ordinal);
```

Comment:
This seems to expect that `flac` is always the last audio codec appended
I'm no expert on the playlist code but that seems like a weird limitation to have?

It's also very possible that I am totally not understanding the contents of `srcPlaylist`

Reply:
We're directly modifying the HLS playlist which only includes one audio codec, so this is works as intended.

Reply:
For future readers this is what is being modified

```
#EXT-X-STREAM-INF:BANDWIDTH=8000000,VIDEO-RANGE=SDR,CODECS="avc1.640033,flac",RESOLUTION=1920x1080,FRAME-RATE=59.94
```